cluster/charts/crossplane/templates/clusterrole.yaml

```diff
@@ -8,14 +8,6 @@ metadata:
     release: {{ .Release.Name }}
     heritage: {{ .Release.Service }}
 rules:
-- apiGroups:
```

Comment:
in reviewing the permissions Crossplane already has, I'm pretty sure it doesn't need permissions for nodes and deployments.